### DISCUSSION TITLE
ETA On this Project

### DISCUSSION THREAD

**Discussion by cosborn-eits** [bilal414/backupsheep#16] [General]
Good morning,

Is there an ETA on this project?  Once BackupSheep stopped working I transitioned to a platform called Weap.IO, but I realized today that half of my servers stopped having snapshots taken of them a few weeks ago without any notification to me...

**Comment by tigerchick** [comment]
I too am very much looking forward to this being ready. In the meantime, you might look at https://simplebackups.com/
I use this for website / database backups but it can schedule server snapshots too. I've found it to be reliable.

**Comment by mrbinayadhikari** [comment]
I’m also looking forward to the update on this project

**Comment by alessandropuricelli** [comment]
Yes please!

**Comment by davferg5000** [comment]
Any update on when this is ready?

**Comment by cosborn-eits** [comment]
Good morning, 

Any updates on this project?

**Reply by bilal414** [reply]
Hi. Been busy with life and some health issues. I will post updates in next few days. I have been working on new project as well https://cloudmoo.com/
